app/presenters/users/dashboard_presenter.rb

```diff
@@ -31,16 +32,15 @@ def issued_certificate_details
         .live
         .includes(:course)
         .map do |issued_certificate|
-          issued_certificate
-            .attributes
-            .slice('id', 'serial_number', 'created_at')
-            .merge(course_name: issued_certificate.course.name)
-        end
+        issued_certificate
+          .attributes
+          .slice('id', 'serial_number', 'created_at')
+          .merge(course_name: issued_certificate.course.name)
+      end
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)